src/core/helpers/page.ts

```diff
@@ -69,6 +68,15 @@ export function createGetServerSideProps(options: CreateGetServerSideProps) {
           "There is a configuration error with this account. Please contact your administrator.",
         );
       }
+      // By default we redirect the user to /workspaces
+      if (ctx.resolvedUrl === "/") {
```

Comment:
From what I see, it will always redirect to /workspaces even if the user does not have the `workspaces` feature flag.

Reply:
Oh okay. I wanted to let the check for the workspaces path to handle it, but I can add that condition too.

Reply:
I think you can try to remove the redirect on the workspaces/index and have only one place where we redirect the user. Otherwise it begins to be a maze where you have to jump from file to file

Reply:
How about adding it to the check at line https://github.com/BLSQ/openhexa-frontend/blob/e0e1ff0c19ae318a0b3a658195ce9238fe87d47b/src/core/helpers/page.ts#L84 ?